### PR TITLE
Add detection for dark theme.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

File: components/Mediator.vue [hunk 1]
```diff
@@ -204,8 +204,42 @@ let resolvePermissionRequest;
...
+    }
+
+    this.relyingDomain = utils.parseUrl(this.relyingOrigin).host;
+
+    // TODO: is this the appropriate place to run this?
+    loadPolyfill(this);
+
+    // attempt to load web app manifest icon
+    const manifest = await getWebAppManifest(this.relyingDomain);
+    this.relyingOriginManifest = manifest;
```
Comment (dlongley) [inline components/Mediator.vue]: There's a lot of other unrelated code being added here... is your history for this branch out of date or something?

Comment (dlongley) [reply components/Mediator.vue]: We might not need *any* changes to this file if the other changes I recommended for default theme handling are added to `getWebAppManifestIcon`.

Comment (JoshDunnDev) [reply components/Mediator.vue]: Not sure to be honest. There was a merge conflict so I tried to resolve it so that is probably why it's like this.
